test/quantization/test_quantize_script.py

```diff
@@ -2089,6 +2089,14 @@ def test_hardswish(self):
             FileCheck().check_not("aten::hardswish") \
                        .run(m.graph)
 
+    def test_elu(self):
+        data = [(torch.rand((1, 2, 5, 5), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
+        elu = torch.nn.ELU()
```

Comment:
this is not enough I think, better to test functionals and inplace variants as well